src/domain/Processors/VehicleProcessor.ts

```diff
@@ -185,10 +205,16 @@ export abstract class VehicleProcessor<T extends Vehicle> {
         new Date().toISOString()
       );
 
-      this.vehicle.techRecord[0].updateType = enums.UPDATE_TYPE.TECH_RECORD_UPDATE;
+      this.vehicle.techRecord[0].updateType =
+        enums.UPDATE_TYPE.TECH_RECORD_UPDATE;
       uniqueRecord.techRecord.push(this.vehicle.techRecord[0]);
       await this.techRecordDAO.updateSingle(uniqueRecord);
-      const mergedRecord = vehiclesFromDB.length > 1 ? this.techRecordsListHandler.mergeRecordsWithSameSystemNumber(vehiclesFromDB) : vehiclesFromDB
+      const mergedRecord =
+        vehiclesFromDB.length > 1
```

Comment:
linting looks a bit strange on this line

Reply:
Not sure why, it's just how the linter did it.